lib/routes.js

```diff
@@ -129,8 +129,23 @@ function addCORS(site) {
  * @returns {express.Router}
  */
 function addSiteController(router, site, providers) {
-  if (site.dir) {
-    const controller = files.tryRequire(site.dir);
+  // fetches itself if it is the main site
+  const parent = siteService.getParentSite(site.slug);
+
+  if (parent && parent.dir) {
+    const controller = files.tryRequire(parent.dir);
+
+    // merge in subsiteController options
+    if (site.subsite) {
+      const subsiteController = files.tryRequire(site.dir);
+
+      // default _.merge overrides values in arrays :facepalm:
+      _.mergeWith(controller, subsiteController, function (par, sub) {
+        if (_.isArray(par) && _.isArray(sub)) {
```

Comment:
We could also use `Array.isArray` which is native JS instead of `lodash`, just like we do in line 151
```suggestion
        if (Array.isArray(par) && Array.isArray(sub)) {
```